parts/k8s/addons/kubernetesmasteraddons-nvidia-device-plugin-daemonset.yaml

```diff
@@ -40,6 +40,13 @@ spec:
       containers:
       - image: <kubernetesNVIDIADevicePluginSpec>
         name: nvidia-device-plugin-ctr
```

Comment:
This name needs to match `NVIDIADevicePluginAddonName`, which should be `nvidia-device-plugin`

Reply:
@sozercan @ritazh what's the action item here? Do we change the const string value, or is this daemonset manifest broken (i.e., change the name here)?

Reply:
i think we can just change the name here to `nvidia-device-plugin`